src/main/java/com/internet/shop/model/Order.java

```diff
@@ -0,0 +1,58 @@
+package com.internet.shop.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Order {
+    private Long id;
+    private List<Product> products;
+    private double price;
+
+    public Order(List<Product> products, double price) {
+        this.products = products;
+        this.price = price;
+    }
+
+    public Long getId() {
```

Comment:
let's either delete lombok dependency from pom.xml or use it

Reply:
deleted...done